src/osemgrep/cli_ci/Ci_CLI.ml

```diff
@@ -81,6 +81,12 @@ contact support@semgrep.com for more information this.|}
   in
   Arg.value (Arg.flag info)
 
+let o_no_secrets_validation : bool Term.t =
+  let info =
+    Arg.info [ "no-secrets-validation" ] ~doc:{|Disables secrets validation|}
```

Comment:
I'm missing where this is implemented. Is it intended to be?

Reply:
This was how I got it to show up in `semgrep ci --help`

<img width="244" alt="Screenshot 2023-10-23 at 7 48 30 PM" src="https://github.com/returntocorp/semgrep/assets/909567/a7aa0ef1-3d3c-43e4-9029-51cc2323aabb">

Reply:
This is going to be misleading, since you've only implemented the part which does anything on the Python side of things. Having the flag in osemgrep too is good, so we don't have to port it later, but it'll be rather confusing if we have it but it is a no-op.

This should also likely be in CLI_Common.ml to line up with where you have it in the Python.